test/jake-util/seeds-recursive/first/third/first.js

```diff
@@ -0,0 +1 @@
+exports.seed = (knex, Promise) => {};
```

Comment:
I think `Promise` is not longer used nor passed. Am I wrong?

Reply:
You're right ;-)